barretenberg/cpp/src/barretenberg/eccvm/eccvm_flavor.hpp

```diff
@@ -920,538 +920,6 @@ class ECCVMFlavor {
 
     using VerifierCommitments = VerifierCommitments_<Commitment, VerificationKey>;
 
```

Comment:
Hopefully we're not giving up to much by getting rid of this. There's a world where we could write failure tests that check every single one of these individual components but its certainly a bit brittle as it is now.